src/mon/OSDMonitor.cc

```diff
@@ -8169,8 +8169,10 @@ int OSDMonitor::prepare_new_pool(string& name,
     pi->use_gmt_hitset = true;
   else
     pi->use_gmt_hitset = false;
-  if (crimson)
+  if (crimson) {
     pi->set_flag(pg_pool_t::FLAG_CRIMSON);
+    pi->set_flag(pg_pool_t::FLAG_NOPGCHANGE);
```

Comment:
OK, I see.

Reply:
Yeah, it's there because crimson can't do splits or merges yet.